lib/alephant/lookup/lookup_cache.rb

```diff
@@ -0,0 +1,67 @@
+require 'dalli-elasticache'
```

Comment:
@samfrench I think I should probably create a new gem of reusable 'modules' that can be `include`'d into different classes.

That way we can do simple things like changing:

```rb
logger.debug "Alephant::LookupCache::#initialize: No config endpoint, NullClient used"
```

to

```rb
logger.debug "#{self.class.name}##{__method__}: No config endpoint, NullClient used"
```

But also try and find clever ways of injecting the differences between two separate gems when including the same module 

e.g. your new `cache.rb` vs the brokers `cache.rb` is subtly different, but very much the same so there is a lot of duplication there and it'll get worse if we need to replicate the `cache.rb` from this PR into the Sequencer as well. So if I moved that code into a reusable 'cache' module, I might be able to then inject the differences via a lambda or some other such mechanism.

I'll discuss with @jfro5 about this. It's likely that this PR stays as you've done it (for now) and I'll get a ticket created for the work to be cleaned up via this new gem.